index.html

```diff
@@ -6,22 +6,40 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta2/css/all.min.css">
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css">
-    <link rel="stylesheet" href="css/style.css">
-    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <title>Portofilio: mobile version</title>
 </head>
 
-<body>
+<body>  
 
-<section>
+   <container id='wraper'>
 
-</section>
-<section>
+     <div class="app">
+      <nav class= 'navbar'>
+       <h1 class='name'>Dejan Vujovic</h1>
+       <a href="https://www.google.com/"><i class="fa-brands fa-creative-commons-nd menu"></i></a>
+     </nav>
+   
 
-</section>
-<section>
-    
-</section>
+     <div class="content">
+  
+     <h2 class='orange'> Hey There. I'm Dejan</h1>
```

Comment:
- [ ] Hi, this looks good, the only thing I want to point out is that according to the requirements this should be an `h1` tag not an `h2`.  also I can see the the closing tag is different from the starting opening tag. please ensure your opening and closing tags always use the same name. 
something like this would be great.
```suggestion
     <h1 class='orange'> Hey There. I'm Dejan</h1>
```